ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/schema_fragments/ip_domain_lookup.schema.yml

```
@@ -0,0 +1,26 @@
+# yaml-language-server: $schema=../../../../plugins/plugin_utils/schema/avd_meta_schema.json
+# Line above is used by RedHat's YAML Schema vscode extension
+# Use Ctrl + Space to get suggestions for every field. Autocomplete will pop up after typing 2 letters.
+type: dict
+keys:
+  ip_domain_lookup:
+    type: dict
+    display_name: Domain Lookup
+    keys:
+      source_interfaces:
+        type: list
+        primary_key: name
+        convert_types:
+        - dict
+        items:
+          type: dict
+          keys:
+            name:
+              type: str
+              required: true
+              description: |
+                Name of Source Interface
```

I used the "name of ..." in the maintenance model, since it referenced to names in other data models. Here it is the primary key in a list called `source_interfaces` so I think we can make it simpler here.
```suggestion
              description: |
                Source Interface
```